framework/PostProcessorFunctions/HStoPSOperator.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on Feb 02, 2018
+@author: alfoa
+"""
+
+#for future compatibility with Python 3--------------------------------------------------------------
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+if not 'xrange' in dir(__builtins__):
+  xrange = range
+#End compatibility block for Python 3----------------------------------------------------------------
+
+#External Modules------------------------------------------------------------------------------------
+import os
+import copy
+import numpy as np
+#External Modules End--------------------------------------------------------------------------------
+
+from PostProcessorInterfaceBaseClass import PostProcessorInterfaceBase
+
+
+class HStoPSOperator(PostProcessorInterfaceBase):
```

Comment:
What does this postprocessor do that is not also done by the HistorySetSnapShot?

Reply:
see above